Readme.md

```diff
@@ -187,10 +187,12 @@ Now let's trying to read and write some data from IoTDB using our Client. To sta
 cd cli/cli
 
 # Unix/OS X
-> $IOTDB_HOME/bin/start-client.sh -h <IP> -p <PORT> -u <USER_NAME>
+> cd /workspace/incubator-iotdb/iotdb-cli/cli/bin/
+> start-client.sh -h <IP> -p <PORT> -u <USER_NAME>
 
 # Windows
-> $IOTDB_HOME\bin\start-client.bat -h <IP> -p <PORT> -u <USER_NAME>
+> cd \workspace\incubator-iotdb\iotdb-cli\cli\bin
+> start-client.bat -h <IP> -p <PORT> -u <USER_NAME>
```

Comment:
I am not sure whether `./` is needed at the head of `start-client.bat` on Win OS.